client/src/components/quiz/components/QuizHandler.jsx

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can we see if we remove the eslint flags, it might be useful to have doc strings on the quizhandler to allow for easier readability?